test/e2e/storage/framework/testdriver.go

```diff
@@ -224,3 +227,23 @@ type VolumeSnapshotStressTestOptions struct {
 	// Number of snapshots to create for each volume.
 	NumSnapshots int
 }
+
+// Metrics to evaluate performance of an operation
+// TODO: Add metrics like median, mode, standard deviation, percentile
+type Metrics struct {
+	AvgLatency time.Duration
+	Throughput float64
+}
+
+// PerformanceTestProvisioningOptions contains parameters for
+// testing provisioning operation performance
+type PerformanceTestProvisioningOptions struct {
+	VolumeSize      string
+	Count           int
+	ExpectedMetrics *Metrics
```

Comment:
What is the operation that gets measured, i.e. "average latency" of what?

I suppose it is "PVC created" till "PVC is bound", but that's worth mentioning here.

Reply:
done!